example-app/src/main/cpp/native-lib.cpp

```diff
@@ -1,9 +1,40 @@
 #include <jni.h>
-#include <string>
 #include <signal.h>
+#include "backtrace-android.h"
 
 extern "C"
+{
 JNIEXPORT void JNICALL
 Java_backtraceio_backtraceio_MainActivity_cppCrash(JNIEnv *env, jobject thiz) {
     raise(SIGSEGV);
+}
+
+JNIEXPORT jboolean JNICALL
+Java_backtraceio_backtraceio_MainActivity_addNativeBreadcrumb(JNIEnv *env, jobject thiz) {
+    std::unordered_map<std::string, std::string> attributes;
+    attributes["C++"] = "true";
+    attributes["Java"] = "false";
```

Comment:
does it make sense to add two attributes instead of one? For example: native:true/false?

Reply:
These are just some dummy attributes to test native breadcrumbs. We won't add anything by default in our real native breadcrumb library `backtrace-android.h` which we expose publicly.

Reply:
I will make that more clear with a comment